### ISSUE TITLE
New site: enable Excel download through API 

### ISSUE THREAD

**Issue by marieALaporte** [bioversity/Crop-Ontology#138]
TD should be downloaded in Excel, not csv (or maybe both)

**Comment by qlands** [comment]
Hi @marieALaporte,

You can enable a download with:

```
from pyramid.response import FileResponse
self.returnRawViewResult = True
response = FileResponse(path_to_the_xlsx_file, request=self.request, content_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
response.content_disposition = ('attachment; filename="file_name.xlsx")
return response
```

**Comment by marieALaporte** [comment]
@qlands is it an issue if the code create a copy of the file on the server? if not, this issue can be closed as it is the download is working. Thanks !

**Comment by marieALaporte** [comment]
@qlands the download seems to not be working on the live server. I got an Internal Server Error. Could you please look into that? Thanks
See for example [here](http://104.155.80.76/report/ontology_id=CO_339)

**Comment by marieALaporte** [comment]
a button on the interface is also needed for users to download the file

**Comment by qlands** [comment]
Fixed in e6ff4dfcf8269020235b3db4dd20ae6e1a72de22

You can download the TD on the main page. Next to each Trait ontology

**Comment by elizabetharnaud** [comment]
Great!
